test/smoke/sentinel.js

```diff
@@ -1,4 +1,35 @@
+const Docker = require('dockerode');
+const getNetworkConfig = require('../../lib/test/getNetworkConfig');
+
+const networkConfig = getNetworkConfig();
+
 describe('Sentinel', () => {
-  // Container should be running and the log should be empty
-  it('should be running without errors');
+  networkConfig.inventory.masternodes.hosts.forEach((nodeName) => {
+    describe(nodeName, () => {
+      it(`should be running without errors ${nodeName}`, async () => {
+        const dockerode = new Docker({
+          host: `http://${networkConfig.inventory._meta.hostvars[nodeName].public_ip}`,
+          port: 2375,
+        });
+        const opts = { filters: '{"name": ["mn_sentinel_sentinel"]}' };
+        const containers = await dockerode.listContainers(opts);
+        expect(containers.length).to.be.equal(1);
+        expect(containers[0].State).to.be.equal('running'); // container is running
+        const containerId = containers[0].Id;
+        await dockerode.getContainer(containerId).logs(
+          {
+            stdout: true,
+            stderr: true,
+            follow: 0,
+            since: (Math.floor(new Date() / 1000) - 300), // logs from last 5 mins
+            timestamps: true,
+          },
+        )
+          .then((error, stream) => {
```

Comment:
why do you need `then`? 

const stream = await dockerode.getContainer(containerId).logs(...)
expect(stream).to.be.undefined();

Reply:
removed